.github/workflows/brew.yml

```
@@ -0,0 +1,47 @@
+name: Brew tests impl
+
+on:
+  workflow_call:
+
+jobs:
+  build_linuxbrew:
+    name: Build on linuxbrew
+    runs-on: ubuntu-latest
+    container:
+      image: homebrew/brew:latest
+
+    steps:
+      # v1 required due to permissions error
+      - name: Checkout mamba repository
+        uses: actions/checkout@v1
+
+      - name: Correct the creation permissions
+        run: sudo chown -R linuxbrew .
+
+      - name: Install prerequisites
```

```suggestion
      - name: Install host and build dependencies
```

I do not have any idea to remove the duplication of this list with [the specifications](https://github.com/mamba-org/mamba/tree/main/dev)'. Let's live with it.